feeds/wifi-ax/mac80211/files/lib/netifd/wireless/mac80211.sh

```diff
@@ -343,6 +348,11 @@ mac80211_hostapd_setup_base() {
 		[ "$rx_stbc" -lt "$cap_rx_stbc" ] && cap_rx_stbc="$rx_stbc"
 		vht_cap="$(( ($vht_cap & ~(0x700)) | ($cap_rx_stbc << 8) ))"
 
+		conf_tx_ant=$(iw phy "$phy" info | grep 'Configured Antennas:' | cut -d: -f2)
+		set -- $conf_tx_ant
+		if [ "$2" == "0x1" ]; then
+			tx_stbc_2by1=0
```

Comment:
Some of the Wifi6 APs seem to use the upper nibble of Configured Antennas instead of the lower nibble.  Example is EX447 (Enis has 1 you can test on).  As a result the test may need to be (addional bracket for order of operation?)
if [ "$2" == "0x1" -o "$2" == "0x10" ];

Reply:
Done.